packages/Tooltip/src/styles.ts

```diff
@@ -55,11 +54,6 @@ export const FadeIn = styled.div<FadeIn>(
     [data-enter] & {
       ${fadeInStyle}
     }
```

Comment:
I think the bug here is that the CSS transition is not being applied to the tooltip element. The tooltip doesn't hide because it's still waiting for a CSS transition/animation to complete.

You can apply the transition directly to the tooltip element or manually call `tooltip.stopAnimation()`:

```jsx
<S.FadeIn onTransitionEnd={tooltip.stopAnimation}>
```

Reply:
@diegohaz Thanks for your answer, I'll check that 🙏🏻